usersession/agent/rest_api.go

```diff
@@ -572,3 +579,80 @@ func postRefreshFinishedNotification(c *Command, r *http.Request) Response {
 	}
 	return SyncResponse(nil)
 }
+
+func collectSnapAppUnits(snapName string, sysd systemd.Systemd) (units []string, err error) {
+	pattern, err := snap.TransientScopeGlob(snapName)
+	if err != nil {
+		return nil, err
+	}
+
+	return sysd.ListUnits(pattern)
+}
+
+func appKill(inst *client.AppInstruction, sysd systemd.Systemd) Response {
+	// TODO: Use inst.Reason as a hint to explain to users what is happening
+	if inst.Signal != 9 {
```

Comment:
Nitpick, any reason not to use `syscall.SIGKILL` here?

Reply:
Good point, updated to use `syscall.Signal` type, Thanks!